src/marlin_stubs/pause/M701_M702.cpp

```diff
@@ -78,6 +80,10 @@ static void load_unload(LoadUnloadMode type, Func f_load_unload, uint32_t min_Z_
         static const float Z_max = get_z_max_pos_mm();
         park_position.z = std::min(std::max(current_position.z, float(min_Z_pos)), Z_max);
     }
+    if (X_pos > 0) {
```

Comment:
use `float `function parameter since it is passed to float anyway
and instead testing `if (X_pos > 0) {` use `isfinite` because 0 is a valid park position